images/ha-config/keepalived/lib/config-generators.sh

```diff
@@ -0,0 +1,268 @@
+#!/bin/bash
```

Comment:
Was what I had from the original demo but it could be anything. Although that said, it does simplify the innards of the sidecar pod to have bash script. I can rework this in a subsequent PR if needed.